elytron/pom.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
   ~ JBoss, Home of Professional Open Source.
-  ~ Copyright 2014 Red Hat, Inc., and individual contributors
+  ~ Copyright 2022 Red Hat, Inc., and individual contributors
```

Comment:
FYI, we are not required to update the year of the license header files. Just if you add a new file, use the expected year but we don't need to update the header when we are modifying the existing ones.

Reply:
Okay. Thank you for the information. I will revert it to the original ones.

Reply:
I think in the past I understood that we needed to update the copyright date for any existing files we modify. Isn't it?

Reply:
Hi @iweiss , you can refer to this Zulip conversation where the copyright headers updating need was discussed: https://wildfly.zulipchat.com/#narrow/stream/174184-wildfly-developers/topic/Copyright.20Headers